lib/core/src/Cardano/Byron/Codec/Cbor.hs

```diff
@@ -593,7 +594,7 @@ decodeUpdateProof = do
 --   passphrases). This is just scheme-specific and is better left out of this
 --   particular function
 encodeAddress :: XPub -> [CBOR.Encoding] -> CBOR.Encoding
-encodeAddress (XPub pub (ChainCode cc)) attrs =
+encodeAddress (CC.XPub pub (CC.ChainCode cc)) attrs =
```

Comment:
The import of `CC` is unnecessary here. This pattern matchin & `pub <> cc` below could be replaced with `xpubToBytes xpub`

Reply:
indeed - done in https://github.com/input-output-hk/cardano-wallet/pull/1640/commits/19cf8232cf85c76e63afd7a10e202abfd7d7eea9